Source/ImperatorWorld.py

```diff
@@ -0,0 +1,32 @@
+from Parser.PyParser import registerKeyword, parseFile
+from Parser.ParserHelpers import getSingleString, getStringList
+
+DateTextOutput = []
+DLCsOutput = []
+ModsOutput = []
+
+
+def getDateStringInList(unused, theStream):
+    DateTextOutput.append(getSingleString(theStream))  # DateTextOutput[0] is a date in a string
+
+
+def getDLCsListInList(unused, theStream):
+    DLCsOutput.append(getStringList(theStream))  # DLCsOutput[0] is a list of DLC strings
+
+
+def getModsListInList(unused, theStream):
+    ModsOutput.append(getStringList(theStream))  # ModsOutput[0] is a list of mod strings
+
+
+def World(ImperatorSaveFileName):
+    print("*** Hello Imperator, loading ImperatorWorld. ***")
+    registerKeyword('date', getDateStringInList)
+    registerKeyword('enabled_dlcs', getDLCsListInList)
+    registerKeyword('enabled_mods', getModsListInList)
+
+    print("Importing Imperator save.")
+    parseFile(ImperatorSaveFileName)
+
+    print('Date is ' + DateTextOutput[0])  # debug
```

Comment:
I'd usually say steer clear of C++ code if you can streamline it, except here. To integrate with Frontend's log display, you need to match current log.txt formatting. So, [date] [debug/warning/info/error] Message.
Create a logger class like we have in Log.txt and feed it data to match our logs.

Reply:
Does the frontent accept only a specific format of time, or it doesn't matter?

Reply:
I would tell you to consult frontend code for that. But beware, that way insanity lies. Many brave men ventured forth and never returned.

Reply:
I found an EU4toVic2 log on my PC, it's YYYY-MM-DD. I think I will just stick to that.

Reply:
I really tried making a class, I swear, but then I saw this:
![image](https://user-images.githubusercontent.com/29546927/78412020-3510be00-7612-11ea-9236-8871b7863822.png)
and decided to run back to my safe space of functional programming 😅 
But the issue should be solved now.

Reply:
Yeah, should be able to. That'll be a pull request (or two, maybe building an exe is its own) of its own though.